kadi/config.py

```diff
@@ -68,6 +68,16 @@ class Conf(ConfigNamespace):
         False, "Include In-work command events that are not yet approved."
     )
 
+    date_start_agasc1p8_earliest = ConfigItem(
+        "2024:203",  # 2024-July-21
```

Comment:
Not required, but might as well bump the dates by a week if we are targeting JUL2924 as the soonest.

Reply:
Done.